### PR TITLE
Add transitive reference to PlatformAbstractions

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

**Comment by moozzyk** [comment]
Why is this needed now?

File: src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
   "dependencies": {
     "Microsoft.Extensions.CommandLineUtils": "1.0.0-*",
+    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-*","
```
Comment (davidfowl) [inline src/Microsoft.AspNetCore.Server.IISIntegration.Tools/project.json]: this is messed up formatting

**Comment by ryanbrandenburg** [comment]
This project references Microsoft.DotNet.ProjectModel, which recently lost PlatformAbstractions as a transitive dependency, but still needs it at build time.

@NTaylorMullen is that explanation any good?

**Comment by davidfowl** [comment]
Where is it used?

**Comment by NTaylorMullen** [comment]
Actually, looks like this project doesn't use platform abstractions. Don't have to do this project :+1:

**Comment by ryanbrandenburg** [comment]
Ok then. Good catch @davidfowl.

**Pull request closed without merging**